ground/src/main/java/com/google/android/ground/model/geometry/Coordinate.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model.geometry
+
+/** A coordinate on a two-dimensional Cartesian plane. */
+data class Coordinate(val x: Double, val y: Double) : Comparable<Coordinate> {
+    companion object {
+        /** Returns the zero coordinate: (0,0) */
+        fun zero() = Coordinate(0.0, 0.0)
```

Comment:
What is this used for?

Reply:
it's not needed anymore, I was going to use it to return default values in some cases instead of throwing exceptions. removed it.